ci/drop-canton.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+set -xeuo pipefail
+
+date=$(date -u "+%Y-%m-%d %H:%M")
+canton_dir=canton
+root_private="/run/user/1000/tmp.W76RmKNhce"
```

Comment:
What is the goal of this directory ? Should you use `root_canton` (temp directory created below) instead ?